langchain/agents/agent_toolkits/pandas/base.py

```diff
@@ -30,6 +30,7 @@ def _get_multi_prompt(
     suffix: Optional[str] = None,
     input_variables: Optional[List[str]] = None,
     include_df_in_prompt: Optional[bool] = True,
+    number_of_head_rows: Optional[int] = 5,
```

Comment:
Optional means this can be None. is that actually true? or should this just be `int`

Reply:
It should be int. The default value for df.head() is 5 _(df.head(x) returns x number of rows from the dataframe)._ So, this way if the value for number_of_head_rows is not passed, the function will perform as it performed earlier with default value. But with this addition, we can also change the value when required.